app/views/collection_show/index.html.erb

```diff
@@ -103,6 +103,16 @@
       <%= render 'search_header' %>
 
       <%= render "document_list", documents: @response.documents %>
+      <% if show_pagination? and @response.total_pages > 1 %>
+        <%# expanded will be shown by CSS at large screen sizes, else compact %>
+        <div class="pagination pagination-alt-expanded">
+          <%= paginate @response, :window => 3, :outer_window => 1, :theme => 'local' %>
+        </div>
+        <div class="pagination pagination-alt-compact">
+          <%= paginate @response, :page_entries_info => page_entries_info(@response), :theme => 'blacklight_compact' %>
+        </div>
+      <% end %>
```

Comment:
If we're gonna be repeating the logic (the conditional, the double pagination for small and big pages) on multiple pages... should we extract it into a partial view template, so it can be re-used without copy and paste? Thoughts @eddierubeiz ?

Reply:
We can totally do this and eliminate the duplication. Probably simpler to do this as a separate PR though, after the two PRs that contain the duplication are merged. I'll create an issue for it.

Reply:
By the way, to answer your earlier question, yes -- the pagination code is already in the featured topics PR.